### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by dscrobonia**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+ - Add css transitions to tool buttons so that the slide out when hovered over.
```
Comment (thc202) [inline CHANGELOG.md]: `the` → `they`?

Comment (kingthorin) [reply CHANGELOG.md]: Good catch!

**Comment by thc202** [comment]
Addressed the comment (and did minor formatting changes).

**Comment by thc202** [comment]
@kingthorin looks good?

**Comment by kingthorin** [comment]
Yup LGTM

**Pull request merged**